docs/docs/Demos.0.DynamicID.md

```diff
@@ -56,78 +59,71 @@ Following best practices is highly encouraged.
 
 #### Spotting the difference in your database
 
-The most obvious way to see the differences between this these two different
-methods of emitting telemetry is by looking and inspecting your database.
+While both of these logging methods result in the same printf() ('flatted') log,
+the one with the Durable ID offers many extra goodies.
```

Comment:
Formatting: The Durable Identifier/ durable identifier / Durable ID - it'd be nice to choose a single term and use it consistently across the doc.